glog/glog.go

```diff
@@ -18,7 +18,7 @@
 // Istio is built on top of zap logger. We depend on some downstream components that use glog for logging.
 // This package makes it so we can intercept the calls to glog and redirect them to zap and thus produce
 // a consistent log for our processes.
-package glog
+package klog
```

Comment:
It this right? Shouldn't this still be glog?

Reply:
Thanks! not sure how that snuck in